### PR TITLE
Update EIP-5656: Move to Last Call

### PR DESCRIPTION
**Pull request by timbeiko**
5656 is part of Dencun, which is going live.

### CODE REVIEW

**Review by jwasinger — APPROVED** [review]

**Review by SamWilsn — CHANGES_REQUESTED** [review]
Please fill out the Security Considerations section. I assume there's very little risk of DoS or memory exhaustion attacks, right?

**Comment by timbeiko** [comment]
cc: @charles-cooper

**Comment by charles-cooper** [comment]
> cc: @charles-cooper

am i able to modify this branch directly?

**Comment by charles-cooper** [comment]
> Please fill out the Security Considerations section. I assume there's very little risk of DoS or memory exhaustion attacks, right?

yes, the analysis for DoS and memory exhaustion is the same as other opcodes which touch memory, as the memory expansion is priced the same.

**Comment by timbeiko** [comment]
> am i able to modify this branch directly?

feel free to create a new one if that's easier, and add the status change as part of the same PR!

File: EIPS/eip-5656.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ title: MCOPY - Memory copying instruction
 description: An efficient EVM instruction for copying memory areas
 author: Alex Beregszaszi (@axic), Paul Dworzanski (@poemm), Jared Wasinger (@jwasinger), Casey Detrio (@cdetrio), Pawel Bylica (@chfast), Charles Cooper (@charles-cooper)
 discussions-to: https://ethereum-magicians.org/t/eip-5656-mcopy-instruction/10890
-status: Review
+status: Last Call
+last-call-deadline: 2024-02-15
```
Comment (timbeiko) [inline EIPS/eip-5656.md:8]:
```suggestion
last-call-deadline: 2024-02-20
```

**Comment by timbeiko** [comment]
@charles-cooper I added a line based on your comment, thanks!

**Comment by charles-cooper** [comment]
we should probably add the additional line to the security considerations:

"Clients should take care that their implementation does *not* use an intermediate buffer (see for instance that the C stdlib `memmove` function does not use an intermediate buffer, per the Notes section of the documentation of [memmove](https://en.cppreference.com/w/cpp/string/byte/memmove), as this is a potential DoS vector. Most language builtins / standard library functions for moving bytes have the correct performance characteristics here."

**Comment by timbeiko** [comment]
Added, thanks @charles-cooper! @SamWilsn, we're ready to merge

File: EIPS/eip-5656.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
+cccccbhbldkgfgvfhrbkedvhrjrebvruhuhubfdvlrhf
```
Comment (timbeiko) [inline EIPS/eip-5656.md:1]:
```suggestion
---
```

**Review by SamWilsn — APPROVED** [review]

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 5656 state from Review to Last Call

**Pull request merged**